tools/update_tool

```diff
@@ -0,0 +1,466 @@
+#!/bin/env python
+
+import argparse
+import json
+import os
+import sys
+import tempfile
+import shlex
```

Comment:
I suggest removing `shlex`, both its uses can be replaced by a simple array.

Reply:
I kinda like having this, it's handy to just copy paste the command and split it without doing it manually, but if you have a strong opinion against shlex, I can remove it.

Reply:
I see what you mean. Alright, you're both against, I'm following the majority, even though I dislike splitting my commands manually. I'll make myself a vim function for that :smile: